### PR TITLE
DCOS-54458 feat(chart): add chart component

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds a minimal chart component which is using react-chartist to create a chart from provided
data. Provided data needs to be in the format `{%label%: %data%}`.

Is part of DCOS-54458

### CODE REVIEW

File: packages/chart/components/Chart.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      labels: Object.keys(data),
+      series: [Object.values(data)]
+    };
+    return (
+      <ChartistGraph
+        type={type}
+        data={graphData}
+        options={{
+          // @ts-ignore
+          fullWidth: true
```
Comment (GeorgiSTodorov) [inline packages/chart/components/Chart.tsx]: Why is this needed?

**Comment by GeorgiSTodorov** [comment]
Looks good, but the last year doesn't fit on the screen.
Screenshot from 2019-05-27 14-38-55

File: packages/chart/components/Chart.tsx (hunk 1 above)
Comment (Poltergeist) [reply packages/chart/components/Chart.tsx]: Because fullWidth is not part of the interface in the options of the library we are using.

File: packages/chart/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default as Chart } from "./components/Chart";
```
Comment (mperrotti) [inline packages/chart/index.ts]: Could we be more specific and call this component `LineChart`?

File: packages/chart/components/LineChart.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+export interface LineChartProps {
+  data: {
+    [label: string]: number;
+  };
+  type: string;
+}
+
+class LineChart extends React.PureComponent<LineChartProps, {}> {
+  public static defaultProps = {
+    type: "Line"
```
Comment (brandonc) [inline packages/chart/components/LineChart.tsx]: Why can you change LineChart to a different chart?

Comment (Poltergeist) [reply packages/chart/components/LineChart.tsx]: Good catch. Forgot to remove thins when I changed this to LineChart.

**Comment by Poltergeist** [comment]
I used chartist since it was the easiest way to create a chart component, we are only providing a very thin wrapper so that we can exchange chartist for something else or even create something ourself at a later point in time. So chartist is not here to stay and we only provide the abstraction which is as universal as possible.
